classes/Combination.php

```diff
@@ -76,34 +76,34 @@ class CombinationCore extends ObjectModel
+            'ean13' => array('type' => self::TYPE_STRING, 'validate' => 'isEan13', 'size' => 13),
+            'isbn' => array('type' => self::TYPE_STRING, 'validate' => 'isIsbn', 'size' => 32),
+            'upc' => array('type' => self::TYPE_STRING, 'validate' => 'isUpc', 'size' => 12),
+            'quantity' => array('type' => self::TYPE_INT, 'validate' => 'isInt', 'size' => 10),
+            'reference' => array('type' => self::TYPE_STRING, 'size' => 32),
             'supplier_reference' => array('type' => self::TYPE_STRING, 'size' => 32),
 
             /* Shop fields */
-            'wholesale_price' =>     array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isPrice', 'size' => 27),
-            'price' =>               array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isNegativePrice', 'size' => 20),
-            'ecotax' =>              array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isPrice', 'size' => 20),
-            'weight' =>              array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isFloat'),
-            'unit_price_impact' =>   array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isNegativePrice', 'size' => 20),
-            'minimal_quantity' =>    array('type' => self::TYPE_INT, 'shop' => true, 'validate' => 'isUnsignedId', 'required' => true),
+            'wholesale_price' => array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isPrice', 'size' => 27),
+            'price' => array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isNegativePrice', 'size' => 20),
+            'ecotax' => array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isPrice', 'size' => 20),
+            'weight' => array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isFloat'),
+            'unit_price_impact' => array('type' => self::TYPE_FLOAT, 'shop' => true, 'validate' => 'isNegativePrice', 'size' => 20),
+            'minimal_quantity' => array('type' => self::TYPE_INT, 'shop' => true, 'validate' => 'isUnsignedId', 'required' => true),
             'low_stock_threshold' => array('type' => self::TYPE_INT, 'shop' => true, 'allow_null' => true, 'validate' => 'isInt'),
-            'low_stock_alert' =>     array('type' => self::TYPE_BOOL, 'shop' => true, 'allow_null' => true, 'validate' => 'isBool'),
-            'default_on' =>          array('type' => self::TYPE_BOOL, 'allow_null' => true, 'shop' => true, 'validate' => 'isBool'),
-            'available_date' =>      array('type' => self::TYPE_DATE, 'shop' => true, 'validate' => 'isDateFormat'),
+            'low_stock_alert' => array('type' => self::TYPE_BOOL, 'shop' => true, 'allow_null' => true, 'validate' => 'isBool'),
+            'default_on' => array('type' => self::TYPE_BOOL, 'allow_null' => true, 'shop' => true, 'validate' => 'isBool'),
+            'available_date' => array('type' => self::TYPE_DATE, 'shop' => true, 'validate' => 'isDateFormat'),
```

Comment:
It's misaligning aligned statements

Reply:
or maybe he's aligning? probably a space/tab conflict :+1:

Reply:
in fact, looking at the class (and not at the diff), it looks cleaner

![clean](https://user-images.githubusercontent.com/1247388/44458007-22946d00-a605-11e8-8261-ea2da3871b35.png)

Reply:
It's a matter of opinion, but I liked it better before:

![screen shot 2018-08-22 at 14 51 51](https://user-images.githubusercontent.com/1009343/44464413-fe438b00-a61a-11e8-8412-39002a5423bd.png)

Reply:
maybe there is an option for it in php cs fixer but we didn't add it

Reply:
Personally prefer the new style, no need IDE configuration and prevent useless add / remove line in git history when creating a new line.